api/v1beta1/mutators.go

```diff
@@ -0,0 +1,33 @@
+package v1beta1
+
+import "strings"
+
+const (
+	CookieMutator = "cookie"
+	HeaderMutator = "header"
+)
+
+type CookieMutatorConfig struct {
+	Cookies map[string]string `json:"cookies"`
+}
+
+func (c *CookieMutatorConfig) HasCookies() bool {
+	return c.Cookies != nil && len(c.Cookies) > 0
+}
+
+func (c *CookieMutatorConfig) ToString() string {
+	var cookies []string
+	for name, value := range c.Cookies {
+		cookies = append(cookies, name+"="+value)
+	}
+
+	return strings.Join(cookies, "; ")
```

Comment:
The RFC specifies that the separator is ";"
```suggestion
	return strings.Join(cookies, ";")
```